### PR TITLE
makes morph spells GC properly

### PR DESCRIPTION
**Pull request by GDNgit**
## What Does This PR Do
makes morph spells GC when a morph is destroyed 

## Why It's Good For The Game
i like morph :) 

## Changelog
NPFC

### CODE REVIEW

**Review by Sirryan2002 — APPROVED** [review]
code looks good

File: code/game/gamemodes/miniantags/morph/morph.dm [hunk 1]
```diff
@@ -65,6 +65,12 @@
 	pass_airlock_spell = new
 	AddSpell(pass_airlock_spell)
 
+/mob/living/simple_animal/hostile/morph/Destroy()
+	. = ..()
```
Comment (Sirryan2002) [inline code/game/gamemodes/miniantags/morph/morph.dm:69]: actually, shouldn't this be after the QDEL_NULLs?

**Review by Sirryan2002 — CHANGES_REQUESTED** [review]

File: code/game/gamemodes/miniantags/morph/morph.dm (hunk 1 above)
Comment (GDNgit) [reply code/game/gamemodes/miniantags/morph/morph.dm:69]: Doesn't matter, either way works. I can just change it to a ``return ..()`` though

Comment (GDNgit) [reply code/game/gamemodes/miniantags/morph/morph.dm]: bam

**Review by Sirryan2002 — APPROVED** [review]

**Review by SteelSlayer — APPROVED** [review]

**Pull request merged**